### PR TITLE
Create LICENSE

### LINKED ISSUE #10: License?
Hi there,

I noticed there's no `LICENSE` file in the repo and none is set in `Cargo.toml` or `pyproject.toml`, so the licensing situation of this project is a bit unclear.

Could you clarify it?

### PR DESCRIPTION
**Pull request by Narsil**
Fix #10

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright [yyyy] [name of copyright owner]
```
Comment (McPatate) [inline LICENSE:189]: Should be hugging face no?

**Review by McPatate — APPROVED** [review]

File: LICENSE (hunk 1 above)
Comment (Narsil) [reply LICENSE:189]: Hmmm maybe.

I think people never read files....

https://github.com/huggingface/transformers/blob/main/LICENSE

....

Comment (Narsil) [inline LICENSE:189]:
```suggestion
   Copyright 2023 HuggingFace Inc.
```

Comment (McPatate) [reply LICENSE]: Shouldn't we open a PR for that? 😅

Comment (Narsil) [reply LICENSE]: I have zero opinion on that. I tend to spend as little time as possible on such things.

Comment (McPatate) [reply LICENSE]: 🤷🏻‍♂️

**Pull request merged**